src/stores/DetailsForm/Store.ts

```diff
@@ -115,12 +115,18 @@ export const getInitialState = (
                 // Run validation on the name. This is done inside the input but
                 //  having the input set custom errors causes issues as we basically
                 //  make two near identical calls to the store and that causes problems.
-                const nameValidation = validateCatalogName(
-                    state.details.data.entityName
+                const NAME_RE = new RegExp(
+                    `^(${PREFIX_NAME_PATTERN}/)+${PREFIX_NAME_PATTERN}$`
                 );
```

Comment:
Please store this regex in the `misc-utils` near the initial pattern. Just trying to make sure it doesn't get ignored/forgotten.